src/create-client.ts

```diff
@@ -187,6 +187,7 @@ export async function createClient(
           if (authenticationResponse) {
             _authkitClientState = "AUTHENTICATED";
             setSessionData(authenticationResponse, { devMode });
+            _onRefresh && _onRefresh(authenticationResponse);
```

Comment:
could consider renaming `onRefresh` to something like `onAuthenticationDataReceived` before we go 1.0, but i think i still like `onRefresh` better (even if it's not technically 100% accurate since we also call this on code exchange).